scripts/create_test_subset.py

```diff
@@ -945,6 +1000,12 @@ def file_exists(path: str) -> bool:
     parser.add_argument(
         '--noninteractive', action='store_true', help='Skip interactive confirmation'
     )
+    parser.add_argument(
+        '--embedded-ids',
```

Comment:
For the CLI argument, I'm wondering if we can call this something like `modify-crams` or `update-headers` to add more clarity to the action. What do you think @jmarshall ?

Reply:
Then the logic becomes,
`if update_headers`
which seems like it makes a little more sense to me

Reply:
Good idea. It's now not just CRAMs and theoretically these IDs could be in not just headers, but `--update-embedded-ids` and `update_embedded_ids` would read better. And the usage message should note that it's on by default.